nav2_amcl/src/amcl_node.cpp

```diff
@@ -331,7 +333,7 @@ AmclNode::requestMap()
 void
 AmclNode::mapReceived(const nav_msgs::msg::OccupancyGrid::SharedPtr msg)
 {
-  RCLCPP_INFO(get_logger(), "AmclNode: A new map was received.");
+  RCLCPP_DEBUG(get_logger(), "AmclNode: A new map was received.");
```

Comment:
Ideally you're right, however due to making the other change which uses the topic instead of the GetMap callback, we now get this message every second also as map is being re-published every second.

Reply:
Got it.
Why's the map being republished every second? That seems like an issue.

Reply:
Because topic latching isn't enabled in ROS2 yet, so it has to re-publish.

Reply:
Should there be a ticket open then along the lines of "when topic latching is implemented in ros2 address map server and amcl changes" ?

Reply:
Good call. I just looked and we may be able to remove this sooner. I filed issue #331